mix_autocomplete.zsh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env zsh
+
+if [[ ! -o interactive ]]; then
+    return
+fi
+
+
+function _mix() {
+  if [[ ! -e "mix.exs" ]]; then
+    return
+  fi
+
+  local mix_md5
+  local completions
+  local current_mix_md5=`md5 -q mix.exs`
```

Comment:
Yup. I noticed that when testing on Ubuntu. It's called md5sum there. Will add a piece of code to find the appropriate tool. In the meantime, I tested it on fresh Ubuntu server install and it hits the cache just fine. Can it be some misconfig on your side?